crates/extensions/c8y_auth_proxy/src/server.rs

```diff
@@ -772,6 +772,7 @@ mod tests {
         panic!("No free port found")
     }
 
+    #[expect(clippy::type_complexity)]
```

Comment:
Nice, I didn't realise `expect` existed!

Reply:
Yup, it was added relatively recently

Reply:
But I have to change it to `#[allow]` because of MSRV 😮‍💨